.storybook/main.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  "stories": [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
These lines need to be adapted refer to the files from your components folder (which you need to create for all of your components like in [this repository](https://github.com/ThomasKirst/react-storybook-example/blob/main/.storybook/main.js)